examples/ConstraintIB/falling_sphere/example.cpp

```diff
@@ -282,7 +282,6 @@ main(int argc, char* argv[])
             for(int structure_idx = 0; structure_idx < structure_COM.size(); structure_idx++){
                 if( (structure_COM[structure_idx][1]-radius) < x_lo[1]){ // changes based on where "open boundaries" are present
```

Comment:
We need to check for the open boundary condition here. This can be checked either as a flag from input file or checking the coefficients of the normal component of velocity.